packages/bpk-tokens/src/web/base/buttons.json

```diff
@@ -100,7 +109,7 @@
       "type": "color"
     },
     "BUTTON_FONT_SIZE": {
-      "value": "{!FONT_SIZE_BASE}",
+      "value": "1.1875rem ",
```

Comment:
We should probably create an alias for this. Something like `BUTTON_FONT_SIZE`

Reply:
Same for the other values in this file...

Reply:
Good shout!